internal/featureflags/featureflags.go

```diff
@@ -188,3 +188,10 @@ func DependencyGraphBasedDeployParallel() FeatureFlag {
 		defaultEnabled: false,
 	}
 }
+
+func Buckets() FeatureFlag {
+	return FeatureFlag{
+		envName:        "MONACO_FEAT_BUCKETS",
```

Comment:
for automation the name is `MONACO_FEAT_AUTOMATION_RESOURCES`
Should we keep that pattern?

Reply:
Good question. I thought about it but then decided against it. 
In automation, we had multiple resources: workflows, business calendars, and scheduling rules.
Here we only have buckets, and thats why I went with only 'MONACO_FEAT_BUCKETS'.